Configuration/AlCa/python/autoCondPhase2.py

```diff
@@ -0,0 +1,80 @@
+# default value for all customizations
+connectionString = "frontier://FrontierProd/CMS_CONDITIONS"
```

Comment:
@mmusich could not we get this directly from the central GlobalTag configuration?

Reply:
how?

Reply:
you mean from here: https://github.com/cms-sw/cmssw/blob/a6191a11f9a39a95324f883b4a35363354d92324/CondCore/ESSources/python/CondDBESSource_cfi.py#L7 ?

Reply:
right,

```
from Configuration.StandardSequences.CondDBESSource_cff import GlobalTag as essource
connectionString = essource.connect.value()
```

should work for what I can see in a quick test of mine

Reply:
Hi @fabiocos, OK. I came to the same conclusion. 
If I change the configuration now (which will make the PR loose all the signature acquired in the last month) will it be sufficient for this to be merged?